src/hardware/ipxserver.cpp

```diff
@@ -144,8 +136,9 @@ static void ackClient(IPaddress clientAddr) {
 	regPacket.maxlen = sizeof(regHeader);
 	regPacket.address = clientAddr;
 	// Send registration string to client.  If client doesn't get this, client will not be registered
-	result = SDLNet_UDP_Send(ipxServerSocket,-1,&regPacket);
-
+	const int result = SDLNet_UDP_Send(ipxServerSocket, -1, &regPacket);
+	if (result == 0)
+		LOG_MSG("IPXSERVER: Connection response not sent");
```

Comment:
Maybe include `%s", SDLNet_GetError()` too, and if it's not much hassle to include `clientAddr`, it's always nice to have specifics.

For example, some users might use a hostname that resolves to a local IP like `192.168.1.4` and immediately know they need a routable IP to traverse the internet.

Reply:
`SDLNet_GetError()` - done.
`clientAddr` - I feel it's out of scope - mostly because SDL_Net seems to be missing a function for converting IPaddress (SDL struct) into a human-readable string.